DOC-2609: The mentions dropdown remained visible even after the menti… (#3579)
tinymce/tinymce-docs: modules/ROOT/pages/7.6.1-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Comments

> The {productname} {release-version} release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.

Reviewer: ```suggestion
The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
```

> **Comments** Comments includes the following fixes.

Reviewer: ```suggestion
**Comments** includes the following fixes.
```

==== The mentions dropdown remained visible even after the mention text was deleted in quick succession.
// #TINY-11673

An issue was identified where the mentions in comments dropdown was still visibile after the mentioned text was removed in quick succession. 

> This behavior occurred because the users list was updated following a successful API fetch request, even when the mentioned text was completely deleted. 
> As a result, users were unable to interact with or close the dropdown.
> 
> In {productname} {release-version}, the logic has been updated to ensure the users list is only refreshed if the mentioned text is not entirely removed. 
> This change ensures that the mentions dropdown automatically disappears when the mentioned text is deleted, improving user experience and functionality.

Reviewer: ```suggestion
This behavior occurred when the users list was updated following a successful API fetch request, even when the mentioned text was completely deleted.

As a result, users were unable to interact with or close the `@` mentions dropdown.

In {productname} {release-version}, the logic has been updated to ensure the users list is only refreshed if the mentioned text is not entirely removed. This change ensures that the mentions dropdown automatically closes when the `@` mentioned text is deleted.
```
